query1.php

```diff
@@ -26,32 +26,41 @@
 		<!-- grid-container class which contains all content using CSS grid -->
 		<div class="grid-container">
 		
-			<div class="nav1">
-				<b><p style = "color: white; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Navigation</p></b>
-			</div>
-		
 			<!-- header class which consists of the header image, text and navigation system -->
 			<div class="header">
+				
 				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Header</p></b>
+				
+				<!-- navigation located at the very top -->		
+				<div class="nav" style = "top: 0vw; position: sticky;">

```

Comment:
Added nav inside the header class for sticky navigation to function appropriately.